src/main/java/duke/exception/ExceptionType.java

```diff
@@ -0,0 +1,19 @@
+package duke.exception;
+
+public enum ExceptionType {
```

Comment:
Very useful enum class so that you Dont Repeat Youself and avoid string literals :) great job